docs/articles/documentation/test-api/obtaining-data-from-the-client/examples-of-using-client-functions.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: docs
+title: Examples of Using Client Functions
+permalink: /documentation/test-api/obtaining-data-from-the-client/examples-of-using-client-functions.html
+---
+# Examples of Using Client Functions
+
+This topic contains some cases of using client functions in tests.
+
+* [Checking a Page URL](#checking-a-page-url)
+* [Obtaining a Browser Alias Within a Test](#obtaining-a-browser-alias-within-a-test)
+* [Injecting External Libraries into a Page from a Test](#injecting-external-libraries-into-a-page-from-a-test)
+* [Accessing Child Nodes in the DOM Hierarchy](#accessing-child-nodes-in-the-dom-hierarchy)
+* [Complex DOM Queries](#complex-dom-queries)
+
+## Checking a Page URL
+
+It is a typical situation when performing some actions on a tested web page (clicking buttons or links) redirects a browser to another page and you need to check whether a desired page is currently opened. For example, you test a login form of your web application and need to verify if a particular web page is opened after a user has logged in.
+
+To return the URL of a page that is currently opened on the client, you can create a client function and use the [window.location.href](https://www.w3schools.com/jsref/prop_loc_href.asp) property within it. Then you can verify if the actual page URL matches the expected one by using the `expect` assertion. The sample below demonstrates how to do this.
```

Comment:
All TestCafe assertions are `expect`. May be better just say 'assertion' or '`contains` assertion method'